File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/iget-training-event-vetting-preview-batches_result.ts



import { GetTrainingEventBatch_Item } from './get-training-event-batch_item';

export interface IGetTrainingEventVettingPreviewBatches_Result {
  
	MaxBatchSize: number;
	LeahyBatchLeadTime: number;
	CourtesyBatchLeadTime: number;
	LeahyBatches?: GetTrainingEventBatch_Item[];
	CourtesyBatches?: GetTrainingEventBatch_Item[];
	LeahyReVettingBatches?: GetTrainingEventBatch_Item[];
	CourtesyReVettingBatches?: GetTrainingEventBatch_Item[];
}

File: GTTS/INL.TrainingService.Models/IGetTrainingEventVettingPreviewBatches_Result.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.TrainingService.Models
{
    public interface IGetTrainingEventVettingPreviewBatches_Result
    {
        int MaxBatchSize { get; set; }
        int LeahyBatchLeadTime { get; set; }
        int CourtesyBatchLeadTime { get; set; }
        List<GetTrainingEventBatch_Item> LeahyBatches { get; set; }
        List<GetTrainingEventBatch_Item> CourtesyBatches { get; set; }
        List<GetTrainingEventBatch_Item> LeahyReVettingBatches { get; set; }
        List<GetTrainingEventBatch_Item> CourtesyReVettingBatches { get; set; }
    }
}
